### PR TITLE
Pluribus Networks admin service module with unit tests

### PR DESCRIPTION
**Pull request by rajaspachipulusu17**
##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME
pn_admin_service.py module with unit tests.

##### ADDITIONAL INFORMATION
ansible 2.4.0.0
  config file = None
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.12 (default, Dec  4 2017, 14:50:18) [GCC 5.4.0 20160609]

```

### CODE REVIEW

**Review by pdam-arista — APPROVED** [review]

**Comment by rajaspachipulusu17** [comment]
@Akasurde can you merge this if you dont see any issue?

File: lib/ansible/modules/network/netvisor/pn_admin_service.py [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+  - This module is used to modify services on the server-switch.
+options:
+  pn_cliswitch:
+    description:
+      - Target switch to run the CLI on.
+    required: False
+    type: str
+  state:
+    description:
+      - State the action to perform. Use 'update' to modify the admin-service.
```
Comment (Akasurde) [inline lib/ansible/modules/network/netvisor/pn_admin_service.py]:
```suggestion
      - State the action to perform. Use C(update) to modify the admin-service.
```

File: lib/ansible/modules/network/netvisor/pn_admin_service.py [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+  description: set of error responses from the admin-service command.
+  returned: on error
+  type: list
+changed:
+  description: indicates whether the CLI caused changes on the target.
+  returned: always
+  type: bool
+"""
+
+import shlex
```
Comment (Akasurde) [inline lib/ansible/modules/network/netvisor/pn_admin_service.py]: Unused import

File: lib/ansible/modules/network/netvisor/pn_admin_service.py [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.network.netvisor.pn_nvos import pn_cli, booleanArgs
```
Comment (Akasurde) [inline lib/ansible/modules/network/netvisor/pn_admin_service.py]:
```suggestion
from ansible.module_utils.network.netvisor.pn_nvos import pn_cli, booleanArgs, run_cli
```

**Review by Akasurde — CHANGES_REQUESTED** [review]
Rest LGTM

**Comment by rajaspachipulusu17** [comment]
@Akasurde can you merge this? if you dont have any issue?

**Comment by Akasurde** [comment]
@rajaspachipulusu17 Could you please fix above mentioned error ?

**Comment by rajaspachipulusu17** [comment]
@Akasurde  i have fixed it

**Review by Akasurde — APPROVED** [review]

**Pull request merged**